pkg/controller/registrysyncer/registrysyncer_test.go

```diff
@@ -164,6 +165,7 @@ func init() {
 }
 
 func TestReconcile(t *testing.T) {
+	t.Parallel()
```

Comment:
if you enable t.Parallel(), you need to copy the `tc` variable in the loop below:
```
tc := tc
```

Because go will reuse the memory of the variable for each loop interation, which means that if the loop body is now in its own goroutine, we modify the `tc` data without waiting for the testcase to finish